Corrected compile errors (#2)
ApryseSDK/webviewer-jquery-sample: README.md

npm start
```

> It will load `http://localhost:3000/`. The application will automatically reload when possible changes occurred to the source files.

Reviewer: ```suggestion
It will load `http://localhost:3000/`. The application will automatically reload if you change any of the source files.
```
I think I like the original phrasing better here
